README.md

```diff
@@ -44,6 +44,25 @@ let s: &CStr = &*s;
 //     - String overhead (3*usize)
 use std::sync::Arc;
 let s = ArcCStr::try_from(format!("foobar")).unwrap();
+// Arc<str>:
+//  + can be created at runtime
```

Comment:
I _think_ a type like this would only work with a string whose contents are defined at compile-time, no? Or can you create an `Arc` whose contents aren't `'static`?

Reply:
It is `impl From<&'_ str> for Arc<str>`, and likewise for `CStr` and slices. It copies from the referenced data into its own allocated storage.

Reply:
I believe it does the same for `Arc<str>: From<String>` and `Arc<[T]>: From<Vec<T>>`; it simply can't reuse the allocated storage.

Reply:
Ah, yes, I suppose that's true!